spec/factories/document_factory.rb

```diff
@@ -4,7 +4,8 @@
   factory :document do
     content_id { SecureRandom.uuid }
     locale { I18n.available_locales.sample }
-    base_path { "/#{SecureRandom.alphanumeric(8)}" }
+    title { SecureRandom.alphanumeric(8) }
```

Comment:
Could we extract these to a trait? Technically it raises the likelihood of false positives for some of the feature tests where we're testing using @document.title.

Reply:
Discussed in person and me and @benthorner agreed to leaving this in. Reasoning being that now we're making use of Title for something (URL path generation) it's good to set a default. At the same time we'd like to keep the factory minimal.